pkg/apis/kops/cluster.go

```diff
@@ -642,6 +642,8 @@ type EtcdManagerSpec struct {
 	Env []EnvVar `json:"env,omitempty"`
 	// BackupInterval which is used for backups. The default is 15 minutes.
 	BackupInterval *metav1.Duration `json:"backupInterval,omitempty"`
+	// BackupRetention which is used for backups. The default is 90 days.
```

Comment:
This doesn't give much information on what the syntax of the string is. Neither do the validation errors.

Reply:
Any suggestion on how to explain it?

Reply:
One idea would be to rename the field to `BackupRetentionDays`. It should 99.99% of the use cases. WDYT?